doc/releases/pacific.rst

```diff
@@ -2,6 +2,48 @@
 Pacific
 =======
 
+v16.2.13 Pacific
+================
+
+This release has all hot-fixes as 16.2.12 plus unintended commits and it was tested, reviewed and approved.
+
+Notable Changes
+---------------
+
+Changelog
+---------
```

Comment:
I think the changelog should include everything starting with 16.2.11.  Otherwise changes that were mistakenly included in 16.2.12 won't be mentioned anywhere at all.

Reply:
It doesn't look like the changelog changed with the latest push, why did you mark this conversation resolved?

Reply:
@idryomov I think I got it now, pls take a look

@ljflores FYI
I reran the release script as `... -r tags/v16.2.11..origin/pacific-release ...

Reply:
I haven't checked cross-checked but this certainly looks more believable!